kubejobs/__init__.py

```diff
@@ -255,7 +255,9 @@ def monitoring_application(self):
             self.LOG.log(("Error: No application found for %s.\
                  %s remaining attempts")
                          % (self.app_id, self.attempts))
-            self.LOG.log(ex.message)
+            self.report_job()
```

Comment:
These function's should not here, they are being called in run function

Reply:
resolved!